src/controllers/auth-controller.js

```diff
@@ -59,15 +59,19 @@ async function register (req, res, next) {
 
     const  passwordHash = await CryptoLib.hashPassword(password);
 
-    await UserModel.create({
-      firstName,
-      lastName,
-      email,
-      password: passwordHash,
-      mobilePhone,
-      address,
-      isActive: true
-    });
+    await Promise.all([
+      UserModel.create({
+        firstName,
+        lastName,
+        email,
+        password: passwordHash,
+        mobilePhone,
+        address,
+        isActive: true
+      }),
+      FavoritesModel.create({}),
```

Comment:
you need to pass user id in this create, so user creation cannot be in same promise.all

same for BucketModel